def try_me():
    return 42